app/assets/javascripts/home.js

```diff
@@ -0,0 +1,2 @@
+// Place all the behaviors and hooks related to the matching controller here.
+// All this logic will automatically be available in application.js.
```

Comment:
You probably should add these files to a gitignore if you don't want them at all.